src/sentry/snuba/metrics/naming_layer/public.py

```diff
@@ -90,3 +94,11 @@ class TransactionStatusTagValue(Enum):
     CANCELLED = "cancelled"
     UNKNOWN = "unknown"
     ABORTED = "aborted"
+
+
+class TransactionSatisfactionTagValue(Enum):
+    """Identifier value for the satisfaction of a transaction."""
+
+    SATISFIED = "satisfied"
+    TOLERATED = "tolerated"
+    FRUSTRATED = "frustrated"
```

Comment:
From ingestion in https://github.com/getsentry/relay/blob/f91625820ebbd6b3103d04aa8f49b86a05321fb0/relay-server/src/metrics_extraction/transactions.rs#L91-L109